Smt/Reconstruct/Real/Polynorm.lean

```diff
@@ -5,7 +5,8 @@ Released under Apache 2.0 license as described in the file LICENSE.
 Authors: Abdalrhman Mohamed
 -/
 
-import Mathlib.Data.Real.Basic
+import Mathlib.Data.Real.Star
```

Comment:
This change is needed as `Real.Basic` as per `4.8.0` has been split up to `Real.Basic` and `Real.Star` as per `4.9.0`. Reverting this results in the following error when tests are run:
```
Test/linarith.lean:146:2: error: [arithPolyNorm]: could not prove x - y = -x + y
```

Reply:
Is the error above the only one caused by the version change? If so, then let's keep `import Mathlib.Data.Real.Basic`. This error message is an instance of a bigger issue that will be resolved with once [cvc5/#10911](https://github.com/cvc5/cvc5/pull/10911) is merged.

Reply:
Yep, this is the only failing test case caused by the version change. I'll revert the change to this import following your suggestion here, thank you.

Also, thank you for mentioning the related cvc5 issue. I found this test case breakage to be rather puzzling, and suspected that there may be a deeper issue involving cvc5's internals.